.github/workflows/deploy-production.yml

```
@@ -0,0 +1,145 @@
+          GHCR_PAT: ${{ secrets.GHCR_PAT }}
+          ENV_B64: ${{ secrets.ENV_FILE_PROD }}
+          IMAGE_TAG: ${{ github.sha }}
+          GHCR_OWNER: ${{ github.repository_owner }}
+          GHCR_REPO: ${{ github.event.repository.name }}
+        run: |
+          ssh ubuntu@37.114.46.226 \
+            "GHCR_PAT='$GHCR_PAT' ENV_B64='$ENV_B64' IMAGE_TAG='$IMAGE_TAG' GHCR_OWNER='$GHCR_OWNER' GHCR_REPO='$GHCR_REPO' bash -s" \
+            < infra/deploy/remote-deploy.sh
+
+      - name: Post-deploy health check
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2034:warning:1:1: i appears unused. Verify use (or export if used externally)

(shellcheck)